docs/codeql/codeql-language-guides/javadoc.rst

```diff
@@ -3,7 +3,9 @@
 Javadoc
 =======
 
-You can use CodeQL to find errors in Javadoc comments in Java code.
+You can use CodeQL to find errors in Javadoc comments in Java/Kotlin code.
```

Comment:
@igfoo is this specific to Java, or does it apply to KDoc too?

Reply:
I think that all the examples rely on JavadocTags, and we don't currently create those for Kotlin code. So leaving this as Java-only for now might be best, but I think it's something we ought to return to before coming out of beta.